spec/requests/admin/admin_holidays_controller_spec.rb

```diff
@@ -65,5 +65,102 @@ module Admin::DiscourseCalendar
         end
       end
     end
+
+    describe "#disable" do
+      context "when the calendar plugin is enabled" do
+        let(:calendar_enabled) { true }
+        let(:dia_de_la_independencia) { { holiday_name: "Día de la Independencia", region_code: "mx" } }
+
+        context "when an admin is signed in" do
+          before do
+            sign_in(admin)
+          end
+
+          it "disables a specific holiday in a specific region" do
+            post "/admin/discourse-calendar/holidays/disable.json",
+              params: {
+                disabled_holiday: dia_de_la_independencia
+              }
+
+            expect(DiscourseCalendar::DisabledHoliday.last.disabled).to eq(true)
+            expect(DiscourseCalendar::DisabledHoliday.last.holiday_name).to eq(dia_de_la_independencia[:holiday_name])
+            expect(DiscourseCalendar::DisabledHoliday.last.region_code).to eq(dia_de_la_independencia[:region_code])
+          end
+
+          context "when a holiday has been added to the calendar" do
+            let(:calendar_post) { create_post(raw: "[calendar]\n[/calendar]") }
+            let(:australia_new_years_day) { { holiday_name: "New Year's Day", date: "2022-01-01", region_code: "au" } }
```

Comment:
```suggestion
            let(:australia_new_years_day) do
              {  
                holiday_name: "New Year's Day", 
                date: "2022-01-01", 
                region_code: "au" 
              }
            end
```

Reply:
I made this change, I also made a few other lines shorter by wrapping them! :+1: